students/ssankura/lesson03/list_lab.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+#Series1
+def series_1():
+	print ("Series1")
+	fruitslist = ["Apples","Pears","Oranges","Peaches"]
+
+	while (True):	
+		response = input("Enter a fruit to add to the list of Fruits > ")
+		if not(response == None and response == ''):
```

Comment:
`if response not in (None, ''):`
Though i don't think you really need input validation on this.